app/Http/Controllers/Api/BiosController.php

```diff
@@ -0,0 +1,25 @@
+<?php namespace Symposium\Http\Controllers\Api;
+
+use Illuminate\Support\Facades\Auth;
+use LucaDegasperi\OAuth2Server\Facades\AuthorizerFacade as Authorizer;
+use Symposium\ApiResources\Bio;
+use User;
+
+class BiosController extends BaseController
+{
+    public function show($id)
+    {
+        $user = User::find(Authorizer::getResourceOwnerId());
+        $bio = $user->bios()->findOrFail($id);
+
+        $bio = new Bio($bio);
+
+        return response()->jsonApi([
+            'data' => [
+                'id' => $bio->getId(),
+                'type' => $bio->getType(),
+                'attributes' => $bio->attributes()
+            ]
+        ]);
+    }
+}
```

Comment:
Does this mean the `/bios` endpoint is always autoscoped to the logged in user? That's an important decision to think about I think. I wonder if `/me/bios` is a more future-proof option, where `/me` will always delegate to `/users/{id}/*`?